### ISSUE TITLE
The validation set result is abnormal

### ISSUE THREAD

**Issue by sipl-zrc** [UCAS-Vincent/UNet3-#1]
Hi, thank you for providing code. I trained on my dataset (contains 700 images) and found that the result of the training set was normal, but the result of the verification set was abnormal.The dice of the verification set is almost 0. I tried a lot of ways to debug it, but it didn't work. So, do you know the probable cause? Thank you!
